doc/user/content/releases/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Releases"
+description: "How Materialize is released"
+disable_list: true
+menu:
+  main:
+    parent: 'about'
+    weight: 10
+---
+
+We are continually improving Materialize with new features and bug fixes. We
+periodically release these improvements to your Materialize account. This page
+describes the changes in each release and the process by which they are
+deployed.
+
+## Release notes
+
+{{< warning >}}
+The v0.27.0 release notes describe the coalesced changes between v0.26 LTS
+and the currently released version of Materialize. We will begin publishing
+weekly release notes after general availability.
+{{< /warning >}}
+
+{{< version-list >}}
+
+For versions that predate cloud-native Materialize, see our
+[historical release notes](https://materialize.com/docs/release-notes/).
+
+## Schedule
+
+We release a new version of Materialize approximately every week. Each
+version includes both new features and bug fixes.
+
+On weeks with a scheduled release, we deploy the release to Materialize accounts
+during the scheduled maintenance window on Wednesday from 4-6pm EST.
```

Comment:
Should this be ET instead of EST so that we don't have to change it when we move to EDT?

Reply:
Good call. Fixed here and in the Notion release process doc.